.gitignore

```diff
@@ -4,3 +4,6 @@ cjs
 esm/api
 node_modules
 /*.tgz
+
+# macOS system files
+.DS_Store
```

Comment:
@hsablonniere If you want me to remove this, I can. I just added it to avoid problems.